### PR TITLE
Support Attribute Selector for CSS Lexer

### PR DESCRIPTION
**Pull request by skoji**
CSS Lexer now support attribute selector like this:
```
a[target="_blank"] { 
    background-color: yellow;
}
```

### CODE REVIEW

File: spec/visual/samples/css [hunk 1]
```diff
@@ -67,3 +67,7 @@ ul#nav li.new {
   unrecognized-prop: 1;
   -moz-prefixed-prop: 2;
 }
+
+a[target="_blank"] { 
+    background-color: yellow;
+}
```
Comment (jneen) [inline spec/visual/samples/css]: Could you add a newline to the end of this file? Thanks!

**Comment by jneen** [comment]
Otherwise, looks great, thanks!

File: spec/visual/samples/css (hunk 1 above)
Comment (skoji) [inline spec/visual/samples/css]: Done. Please check the change.

**Comment by jneen** [comment]
Released in v1.11.0.

**Pull request merged**